File: src/test/resources/expectedfolder/admin/LoginResponse.cs
using System;
using System.Collections.Generic;
using com.google.devtools.java2csharp.testclasses;
using com.google.devtools.java2csharp.testclasses.admin;

namespace com.google.devtools.java2csharp.testclasses.admin
{
	public class LoginResponse : Message
	{
		private ResponseContent content;

		private Information information;

		private IList<ContentArea> areas;

		protected internal LoginResponse()
			: base()
		{
		}

		public LoginResponse(Header header, ResponseContent content, Information information
			, IList<ContentArea> areas)
			: base(header)
		{
			this.content = content;
			this.information = information;
			this.areas = areas;
		}

		public virtual ResponseContent Content
		{
			get
			{
				return content;
			}
			set
			{
				ResponseContent content = value;
				this.content = content;
			}
		}

		public virtual Information Information
		{
			get
			{
				return information;
			}
			set
			{
				Information information = value;
				this.information = information;
			}
		}

		public virtual IList<ContentArea> Areas
		{
			get
			{
				return areas;
			}
			set
			{
				IList<ContentArea> areas = value;
				this.areas = areas;
			}
		}
	}
}

File: src/test/java/com/google/devtools/java2csharp/testclasses/admin/LoginResponse.java
package com.google.devtools.java2csharp.testclasses.admin;

import java.util.List;

import com.google.devtools.java2csharp.testclasses.Header;
import com.google.devtools.java2csharp.testclasses.ResponseContent;

public class LoginResponse extends Message {
	private ResponseContent content;

	private Information information;

	private List<ContentArea> areas;

	protected LoginResponse() {
		super();
	}

	public LoginResponse(Header header,
			ResponseContent content, Information information,
			List<ContentArea> areas) {
		super(header);
		this.content = content;
		this.information = information;
		this.areas = areas;
	}

	public ResponseContent getContent() {
		return content;
	}

	public void setContent(ResponseContent content) {
		this.content = content;
	}

	public Information getInformation() {
		return information;
	}

	public void setInformation(Information information) {
		this.information = information;
	}

	public List<ContentArea> getAreas() {
		return areas;
	}

	public void setAreas(List<ContentArea> areas) {
		this.areas = areas;
	}
}
